controllers/common/views/information-page.njk

```diff
@@ -8,46 +8,23 @@
 {% if not pageHero.image %}{% set bodyClass = 'has-static-header' %}{% endif %}
 
 {% block content %}
-    {% macro primaryContent() %}
-        {{ breadcrumbTrail(breadcrumbs) }}
-        <div class="s-prose accent-content--{{ pageAccent }} u-constrained-content-wide js-annotate-links">
-            {% if not pageHero.image %}<h1>{{ content.title }}</h1>{% endif %}
-            {{ content.introduction | safe }}
-        </div>
-        {{ segmentLinks(customSegmentLinks | default(content.segments)) }}
-    {% endmacro %}
-
     <main id="content">
         {% if pageHero.image %}
-            {{ hero(
-                titleText = title,
-                image = pageHero.image,
-                accent = pageAccent
-            ) }}
+            {{ hero(title, pageHero.image) }}
         {% endif %}
 
         {% if pageHero.image %}<div class="nudge-up">{% endif %}
 
         {% block contentPrimary %}
-            {% if content.relatedContent %}
```

Comment:
By looking at the Field Inventory plugin in Craft I noticed that a) this `relatedContent` field is only included on BBO pages an b) it's never filled in. I _think_ this is a hold over from an old version of the people pages. Basically this can go and the field can be removed from the API.

Reply:
Excellent!